%=======================================================================================%
\pagenumbering{arabic}
\chapter{Introduction}
\setcounter{page}{1}
\label{chap:intro}

\section{Life Through a Microscopic Lens - Molecular Biophysics}
\chapquote{``All science is either physics or stamp collecting."}{Ernest Rutherford}

\vskip 0.5cm

Life, one of the greatest mysteries in the universe we live in, fascinates humankind since 
the dawn of time. Over the millennia, attempts to explain observations in nature were shaped 
by a macroscopic world-view (i.e. a top-down approach). From the time of Aristotle, theories 
in biology (or science in general) are formed by data collection obtained from systematic 
observation at the macroscopic scale. The atomistic view of Democritus\footnote{a view based 
on \textit{reductionism}} was rejected, and the Aristotelian school of thought became the 
prevailing view. This orthodox view started to crack during the scientific revolution in the 
enlightenment period and more so as a result of the industrial revolution. Primarily, the 
invention of steam engines produced the field of \textit{thermodynamics}, which ultimately 
led to the development of \textit{statistical mechanics}. This field of physics, developed by 
physicists like Boltzmann, Maxwell and Gibbs, demonstrates that the observed phenomenon at the 
macroscopic scale can be explained by the behaviour of particles at the microscopic scale. 
Coupled with the development of quantum mechanics, the atomic theory is the foundation behind 
modern chemistry, condensed matter physics and molecular biology. The use of physics to study 
life at the microscopic scale is referred to as the interdisciplinary field of \textit{molecular 
biophysics}.

The understanding of biomolecular processes is the goal of molecular biophysics. However, the 
field is not limited to just understanding biomolecular processes. The understanding gained 
from studying such processes at the microscale can also be useful in biomolecular engineering 
and drug design. Following the philosophy of statistical mechanics, the field 
tries to understand or explain what we observe experimentally with the processes that occur at 
the molecular scale. Biological functions can be explained by its molecular structure, dynamical 
behaviour or ligand binding. Historically, research in this field is done with experimental methods 
and techniques. For example, ligand binding in ion channels can be probed using electrophysiological 
experiments. In such experiments, the voltage and currents of ions/molecules passing through 
the channel are measured. The number of molecules passing through a channel is used to measure 
its conductance. The binding free energy is determined from the ligand concentration (inhibitor) 
where the conductance of the channel drops to 50\%, known as IC$_{50}$ (provided the inhibitor 
completely inhibits the transport). However, the dynamics and mechanics of the binding process 
is not known as experiments can only indicate the strength of binding. The possible ligand 
binding site can be probed by performing electrophysiological experiments on the mutant species 
(i.e., by mutating specific residues). Carrying out such experiments can indicate if the conductance 
is impaired (or enhanced) and thus can tell us which residues are (maybe) involved in the binding 
process. However, the picture we can draw from these experiments is vague in regards to a molecular 
description. Molecular structures are determined by experiments like X-ray diffraction, neutron 
diffraction, nuclear magnetic resonance (NMR) and more recently cryo-electron microscopy. With 
these methods, we can obtain snapshots at different states, but again they do not tell us anything 
about the dynamics. Since experiments are limited in their ability to probe the dynamics of 
biomolecules, we can use the computer as a \textit{computational microscopy} that complements 
physical observations.

\section{Computational Molecular Biophysics}

\subsection{The Rise of Computers}
\chapquote{``Computers make it easier to do a lot of things, but most of the 
things they make it easier to do don't need to be done."} {Andy Rooney}

\vskip 0.5cm

Computers, we see them and use them, whether it is a hand-held mobile phone, tablets, laptops, 
and desktops, to name a few. First world economies rely heavily on computers to the point where 
if all computers were to stop working the economy would plunge instantly. Our lives have become 
intertwined with the use of computers where it is now tough to survive in today's world without 
having knowledge and skill of using computers. However, it was not too long ago when computers 
were not even relevant to our lives. Computers were initially simple devices used to perform 
arithmetic calculations like the abacus that has been in use since ancient times going back to 
the Sumerian civilisation. Charles Babbage invented the first mechanical computers during the 
industrial revolution, which uses punch cards as the input and output. Electronic computers, on 
the other hand, came about in the 20th century and were primarily motivated by military research 
with the first built by John V. Atanasoff and Clifford Berry in 1937. This computer and the 
subsequent versions used vacuum tubes and limited in its use. These machines are classified as 
the first generation of computers that uses vacuum tubes. The second generation came with the 
invention of the transistors with the Universal Automatic Computer (UNIVAC) being the first 
commercially available. Third-generation computers are based on integrated circuits (IC), 
which is the foundation of all electronics today. However, running molecular simulations on 
computers would be limited to small and simple systems if it were not for the invention of 
high-performance computing (HPC) that provided the hardware for parallelisation. HPC enables 
larger systems to be studied that were deemed impossible before. More recently, the use of 
the graphical processing unit (GPU) in calculations reduces the computation time by a 
significant factor that newer supercomputers are built with them (e.g., the recently install 
Summit computer in Oak Ridge). GPUs not only enables faster computation but gives the power 
and speed of HPCs to the masses. Coupled with open-source software, anyone with a 
desktop computer can, in principle, run simulations and calculations that were once only 
accessible to the military or elite academic institutions. Although most are not likely to 
use their personal computers for scientific purposes, this decentralisation of computing 
power is a valuable opportunity for scientific progress, and the scientific community should 
take advantage of this opportunity.

\begin{figure}[t!]
\centering
\includegraphics[width=12.5cm]{Figures/Intro/Computer-Models.png}
\caption{Computer simulations and the connection to experiment and theory in science. 
(This flow chart is a modified version from Allen and Tildesley~\cite{Allen1987} designed 
in Tikz)}
\label{intro:models}
\end{figure}

\pagebreak
\subsection{Computer Modelling}
\chapquote{``Part of the inhumanity of the computer is that, once it is
competently programmed and working smoothly, it is completely honest."}{Isaac Asimov}

\vskip 0.5cm

\begin{figure}[b!]
\begin{minipage}{\textwidth}
\centering
\includegraphics[width=12.5cm]{Figures/Intro/Multiscale-Modelling.jpg}
\caption{Multi-scale modelling of microscopic processes, specifically for biomolecules, 
with time and length in the vertical and horizontal axes, respectively. (Image obtained 
from the Rhadhakrishnan lab, Ref.~\cite{Radhakrishnan2018})}
\label{intro:mscale}
\end{minipage}
\end{figure}

% Description of Computer modelling compared to experiment and theory
In experiments, we are interested in seeing some signal change of a system when its 
environment is perturbed. By doing this, we can gain some insights into the behaviour of 
the system. In theory, we try to use the laws of physics/nature and mathematics to model 
what we observe in experiments. However, only some problems have analytical solutions. 
For example, in quantum mechanics, the only analytical solution we know of is the hydrogen 
atom. Describing larger atoms requires approximations to the theory. Thus, because of this 
limitation, there may be a significant gap in the connection between experiments and theory. 
On this end, computer simulations attempt to bridge the gap between experiments and theory 
(see \figref{intro:models}). In computer simulations, real systems are modelled based on the 
available theory. The model is then solved using numerical (approximate) methods and converted 
into an algorithm that can be understood by a computer. The results from computer simulations 
can be compared to both experimental results and theoretical predictions. Comparing 
simulations with experiments determines the accuracy of the computer model while comparing 
with theoretical predictions tests the validity of the theoretical models itself. Thus, 
computer simulation is a tool that can bridge the gap between experiments and theory. Several 
different computational models can be used to simulate biological processes. 
The choice of model depends on the scale of the system, spatially and temporally, and this is 
known in the literature as multi-scale modelling, as shown in \figref{intro:mscale}. For example, 
on a large scale, simulating neuron to neuron signalling requires neural network algorithms. 
However, this does not capture any microscopic details. On a small scale, quantum mechanics 
can describe chemical reactions accurately albeit, the computation becomes very expensive. 
Thus, there is a trade-off between accuracy and computation time.

\subsection{Simulations of Biomolecules}
\chapquote{``Are you living in a computer simulation?"}{Nick Bostrom}

\vskip 0.5cm

Today, the ``\textit{standard model}" for modelling the dynamics of biological systems is 
molecular dynamics (MD). The first-ever MD simulation was performed by Berni Alder and Thomas 
Wainwright in 1957~\cite{Alder1959}. They simulated elastic collisions of hard-sphere particles 
on the IBM 704 computer and was inspired by the earlier success of the Metropolis Monte Carlo 
(MMC) simulation in 1953~\cite{Metropolis1953}. However, the breakthrough came when Rahman 
performed simulations of Argon molecules using the Lennard-Jones (LJ) potential in 
1964~\cite{Rahman1964} and water molecules in 1971~\cite{Rahman1971}. This provided a stepping 
stone for biomolecular simulations as almost all simulations today uses the LJ potential. 
Following the work of Rahman, the first MD simulation of small proteins was performed in the 
1970s by Harvard scientists Andrew McCammon, Bruce Gelin and Martin Karplus~\cite{McCammon1977}. 
They simulated the bovine pancreatic trypsin inhibitor (BPTI) globular protein for a total of 
9.2~ps. The simulation study demonstrated that protein structures are not rigid but flexible 
and play a role in protein function. This motivated the development of more accurate force fields 
(\secref{sec:forcefield}) that best represent experiments. There are now many different force 
fields available for biomolecules with the most commonly used packages includes AMBER (Assisted 
Model Building and Energy)~\cite{Maier2015}, CHARMM (Chemistry at Harvard Macromolecular 
Mechanics)~\cite{MacKerell1998}, GROMACS (GROningen Machine for Chemical 
Simulation)~\cite{Oostenbrink2004} and OPLS (Optimized Potential for Liquid  
Simulations)~\cite{Robertson2015}. MD simulations also require the structure of biomolecules. 
These can be obtained from NMR or crystallography that are deposited in the protein data bank. 
In addition to basic MD simulations of biomolecules, advanced free energy methods have been 
developed that are used to compare computed quantities with experimental data (see 
\secref{sec:freenergy} for more details). This thesis reports on the investigation of three 
different systems: glutamate transporters, gramicidin A channel and ion solvation in water, 
using free energy methods.

\section{Glutamate Transporters}
\begin{figure}[b!]
\centering
\includegraphics[width=11cm]{Figures/Intro/Neuron.jpg}
\caption{Schematic diagram of the synapse between two neurons. 
(Image obtained from Ref.~\cite{Splettstoesser2017})}
\label{intro:neuron}
\end{figure}

Nerve impulses occur when neurotransmitters are sent from one neuron to another, creating a 
domino effect. Neurotransmitters are stored in vesicles and transported across the axon. When 
it reaches the axon terminal, the vesicle fuses with the lipid cell membrane, which in turn 
releases the neurotransmitter into the synaptic cleft. The neurotransmitters diffuse around in 
the synaptic cleft, and a small percentage is picked up by ionotropic receptors on the dendrite. 
The neurotransmitter molecules picked up by the receptors are deactivated and released back 
into the synaptic cleft. As part of normal regulation, the neurotransmitters need to be cleaned 
up from the synaptic cleft. The clean up is performed by transport proteins located at the 
axon terminal. These transport proteins essentially recycle neurotransmitters back into the 
axon, and the transport process is coupled with ions. \figref{intro:neuron} illustrates the 
process of sending neurotransmitter between two neurons.

\begin{figure}[t!]
\centering
\includegraphics[width=0.6\textwidth]{Figures/Intro/Glt-Transport.jpg}
\caption{(A) Transport cycle of glutamate transporter depicted by the alternating 
access model of glutamate transporter. (B) Kinetic model of ligand binding in EAATs. 
The protein releases \K\ for three \Na, \Hi\ and the substrate (S). During substrate 
translocation, a channel pore opens that allows \Cl\ conductance. The protein releases 
all ligands and captures another \K\ that completes the cycle. (The figure is adapted 
from Ref.~\cite{Vandenberg2013})}
\label{intro:altmodel}
\end{figure}

L-glutamate is the major excitatory neurotransmitter in the human central nervous 
system~\cite{Danbolt2001}. Since glutamic acid is a charged molecule, the excess of glutamate 
leads to over-activation of receptors. Over-activation can lead to neuron damage, also known as 
excitotoxicity. It is known that excitotoxicity is linked to Alzheimerâ€™s disease~\cite{Hynd2004}, 
amyotrophic lateral sclerosis~\cite{Rothstein1992}, ischaemia~\cite{Rossi2000} and 
epilepsy~\cite{During1993}. The transport protein responsible for regulating glutamate is called 
the excitatory amino acid transporters (EAATs)~\cite{Danbolt2001}. There are five different EAATs 
in total, and they are part of the solute carrier family A (SLC1) of transporters that includes 
two neutral amino-acid transporters (ASCT)~\cite{Arriza1993}. Glutamate in EAATs is coupled to 
the co-transport of three \Na\ and one \Hi\ followed by the counter-transport of 
\K~\cite{Levy1998,Owe2006}. In addition to these ions, EAATs is known to allow the transport of 
\Cl~\cite{Fairman1995}. EAAT4-5 are known to behave as an ion channel for \Cl\ while this activity 
is smaller in EAAT1-3. A model for the transport of glutamate is shown in \figrefi{intro:altmodel}{A} 
and is based on the alternating access model proposed by Jardetsky in 1966~\cite{Jardetzky1966}. 
To put it simply, glutamate binds to the transporter in the outward-facing conformation along with 
the coupled ions. The gate closes and the protein undergoes conformational changes that transport 
the ligands across the membrane bilayer. Once the gate is exposed to the intracellular media, 
the ligands are released, a \K\ ion comes in and further conformational changes takes place to 
reorient the protein back to the extracellular side completing the cycle. During the conformational 
change, a pathway is opened that allows for \Cl\ conductance. \figrefi{intro:altmodel}{B} 
illustrates the kinetic model of the transport mechanism in EAATs.

\begin{figure}[b!]
\centering
\includegraphics[width=0.55\textwidth]{Figures/Intro/GltPh-Topology.jpg}
\caption{A 2D representation of the topology of a single \GltPh\ chain. 
(The figure is adapted from Ref.~\cite{Vandenberg2013})}
\label{intro:glttop}
\end{figure}

Breakthroughs in studying membrane proteins came from x-ray crystallographic experiments, 
which provided a 3D structure of the proteins. For glutamate transporters, this came in 
2004~\cite{Yernool2004} with the crystal structure of the prokaryotic homologue of EAATs from 
\textit{Pyrococcus horikoshii} called \GltPh. The topology of \GltPh\ (\figref{intro:glttop}) 
consists of eight transmembrane helices, TM1-8, and two hairpin loops, HP1-2. X-ray crystallography 
reveals that \GltPh\ is made up of three identical subunits held together by non-covalent bonds 
in a trimer formation. \figrefi{intro:gltph}{A} shows \GltPh\ in the outward-facing conformation 
with a 50~\angs\ wide bowl-shaped  trimer and 30~\angs\ in depth.\GltPh\ shares a 36\% sequence 
identity to the mammalian EAATs but the sequence in the binding pocket is close to 
60\%~\cite{Kavanaugh1997}. The most important part of the binding pocket is the highly conserved 
non-helical NMDGT motif between TM7a and TM7b. This motif plays a crucial role in ligand binding 
in all SLC1 family. \GltPh\ also differs from EAATs in that it is only selective to aspartic acid 
and does not require the co-transport of \Hi\ and counter-transport of \K. Although there are 
differences with EAATs, \GltPh\ is a good candidate as a model for the structure and function of 
EAATs. 

\begin{figure}[t!]
\centering
\includegraphics[width=1.0\textwidth]{Figures/Intro/Glutamate-Transporter.jpg}
\caption{Molecular structure of \GltPh: (A) side and top view embedded in a lipid bilayer. (B) 
\GltPh\ structure from the (left) outward to (right) inward conformation. The blue and yellow 
region represents the trimerisation and transport domain, respectively. (C) Fully bound \GltPh, 
including three \Na\ and aspartate. The Na3 site is shown on the right coordinated by the D312 
residue. (All figures are created using VMD based on the PDB ID: 2NWX crystal structure)}
\label{intro:gltph}
\end{figure}

The \GltPh\ structure is grouped into two domains: trimerisation and transport domains. TM1, 2, 4 
and 5 belong to the trimerisation domain while TM3, 6, 7 and 8 and the hairpin loops belong to the 
transport domain. As shown in the crystal structure, the two hairpin loops HP1-HP2 forms a gate 
that locks the ligand inside the protein. During substrate translocation across the membrane, the 
transport domain undergoes large conformational change takes place so that the HP1-HP2 gate is 
exposed to the intracellular medium. The two states are shown in \figrefi{intro:gltph}{B}, which 
illustrate the conformational change of the transport domain across the bilayer. Computational 
studies with the anisotropic network model (ANM) shows that the transition of each protomer occurs 
independently~\cite{Jiang2011,Bahar2010}. Also, the transport and trimerisation domains move in the 
opposite direction with respect to the membrane normal~\cite{Stolzenberg2012}. This behaviour is 
observed in FRET imaging experiments that indicate a stochastic independent movement of each 
subunit~\cite{Akyuz2013,Akyuz2015}. The 2007 crystal structure (PDB: 2NWX)~\cite{Boudker2007} 
identified two \Na\ ions (dubbed as Na1 and Na2) and the aspartic acid, \figrefi{intro:gltph}{C}. 
This crystal structure, however, did not reveal the binding site of the third \Na\ ion (Na3). 
The use of \Tl\ ions instead of \Na\ was thought to be the cause of this since the ionic radius 
of \Tl\ ion is 0.5~\angs\ than \Na. If the binding site is deeply buried inside the protein, then 
ion may not be able to make its way to the site. Many computational studies were carried out to 
identify possible binding sites. However, the best site is the one proposed in 
Ref.~\cite{Bastug2012}, as shown in \figrefi{intro:gltph}{C}. This site is supported by mutagenesis
experiments~\cite{Bastug2012} and a recent crystal structure of \GltTk\ (a close cousin of \GltPh) 
reveals the same binding site that uses \Na\ instead of \Tl~\cite{Guskov2016}.

The major portion of this thesis deals with investigating the ligand binding in \GltPh\ further 
than what has been done in the literature and the review given in this section is brief. A more 
detailed review is given in \chapref{chap:gltph}, where the problems encountered in simulations 
are discussed. In particular, the review focuses on the insights 
gained on the transport mechanism through computational methods, which otherwise is not directly 
accessible by experimental probes. Briefly, this thesis reports on three major findings in \GltPh: 
(i) the elucidation of the Na2 site, (ii) resolution of the ligand-binding paradox and (iii) 
details the conformational changes upon the release of Na3.

%One problem investigated is the apparent discrepancy between simulations 
%and experiments about the Na2 site. In almost all simulations the \Na\ ion leaves the binding site within 
%a short simulation 
%time~\cite{DeChancie2011a,Venkatesan2015,Heinzelmann2013,Heinzelmann2011,Heinzelmann2014a}. 
%The ion was, however, stable in a binding site that is shifted by $\sim$2~\angs\ away from the Na2 site. We 
%trace the root cause of this discrepancy as a result the force field parameters for the methionine residue M311
%in \GltPh\ and M314 in \GltTk. In addition, the ion and substrate binding in \GltPh\ is investigated in more 
%detail. This investigation resolves the paradoxical issue observed in experiments where the binding of the 
%substrate is fast with high-affinity while \Na\ binding is slow with a low-affinity. Finally, the unbinding process 
%of \Na\ from Na3 site in the inward-facing conformation is studied. The escape of \Na\ reveals structural 
%changes taking place as the ion leaves the binding site.

%Several sites were proposed~\cite{Tao2010,Huang2010,Holley2009,Larsson2010,Bastug2012}, but the most promising was 
%the site that involves the D312 residue as part of the coordination. From a physics point of view, there 
%cannot be an unpaired charge inside the protein similar to why metals have charges on the surface but not 
%inside the bulk material. Mutagenesis experiments of EAAT3 indicate residues T92 and D312 are involved in 
%\Na\ binding since it impairs substrate transport~\cite{Tao2010,Tao2006}. Also, since these residues do 
%not coordinate Na1 and Na2, they must be involved in the Na3 coordination. MD simulations without Na3 shows 
%that D312 flips to coordinate with Na1 forming an intermediate binding site labelled as Na1$^{\prime}$. 
%Two papers proposed Na3 sites coordinated by D312, one by Huang and 
%Tajkhorshid~\cite{Huang2008,Huang2010} and the other by our group in the University of 
%Sydney~\cite{Bastug2012}. The difference between the two proposed sites is the coordination of Na3 by the 
%S93 residue. Mutagenesis experiments on S93A shows that the proposed site from our group is the most 
%likely candidate~\cite{Bastug2012}. This proposed site is verified by the recent crystal of \GltTk\ which 
%was resolved with \Na\ instead of \Tl~\cite{Guskov2016}. The \Na\ is coordinated by the residues proposed 
%from MD simulations and mutagenesis experiments, see \figrefi{intro:gltph}{C}. This demonstrates the 
%strength of MD simulations coupled with experiments to make predictions in molecular biology. Subsequent 
%computational studies investigated the ligand binding order of both outward- and inward-facing 
%conformation~\cite{Heinzelmann2013,Heinzelmann2011}. In addition to computational studies of \GltPh\, 
%homology models of EAAT3 based on the \GltPh\ structure were made and the potential sites for \Hi\ and 
%\K\ were probed~\cite{Heinzelmann2014,Heinzelmann2014a,Holley2009}. The review of the computational 
%studies of glutamate transporters given here is a brief introduction and a more detailed review is 
%available in Ref.~\cite{Setiadi2015}.

\section{Gramicidin A Ion Channel}
Gramicidin is an antibiotic peptide consisting of three mixtures: A, B and C. The most common 
compound in nature is gramicidin A (gA) and are produced and obtained from the soil bacterium 
\textit{Bacillus brevis}. gA disrupts bacteria by allowing ions to flow into the plasma membrane 
causing an imbalance of ion concentration with a conductance rate of ${\sim}10^7$ ions per second. 
As a peptide, it is made up of alternating L- and D-amino acids with a total of 16 residues. gA 
first enters the plasma membrane as a monomer. In the membrane, the monomer diffuses until it 
finds another monomer. On close proximity, the two monomers lock on to form a dimer creating a 
water-filled pore thereby allowing ion conductance. This dimer formation was first proposed by Dan 
Urry in 1971~\cite{Urry1971} and \figref{intro:gAIntro} illustrates the process by which an ion 
enters the plasma membrane. The ion, in this example a \K\ ion, will diffuse through the pore 
formed by gA with water molecules aligned in a single-file manner. This was the first antibiotic 
used clinically and was discovered by Ren\'{e} Dubos in 1939.

\begin{figure}[b!]
\centering
\includegraphics[width=11cm]{Figures/Intro/gA-Intro.png}
\caption{(A) Gramicidin A ion channel monomers are diffusing in the membrane bilayer and (B) 
dimer formation opens a channel for \K\ to pass through the bilayer. The formation of a 
dimer causes the membrane bilayer to bend and compress, reducing the thickness around the 
dimer (bilayer image is obtained from Ref.~\cite{Lundbaek2010}, and \K\ ion is added in 
Tikz). (C) A snapshot of gramicidin A showing \K\ and the single-file water molecules. 
(gA cartoon representation is created in VMD)}
\label{intro:gAIntro}
\end{figure}

Computational studies use gA structures resolved with solid-state nuclear magnetic resonance 
(NMR). The first of which is PDB: 1MAG~\cite{Ketchem1996} in 1997, followed by PDB: 
1JNO~\cite{Townsley2001} in 2001 where gA is solvated in sodium dodecyl sulphate (SDS) micelles. 
MD simulations show that the 1MAG structure eventually matches the conformation of the 1JNO 
structure~\cite{Allen2003a}. There are now many atomic structures of gA with itself or in complex 
with other ions and molecules that are available in the protein data bank. Due to its simple 
structure and being the first ion channel to have its structure resolved, gA is sometimes used 
as a model for membrane proteins. For example, gA was used as a starting point to study potassium 
channels before these channels were crystallised. This reductionist approach gave the motivation 
for a plethora of computational investigations of ion conductance in biological systems. The 
validity of using gA to model ion channels is partially endorsed when the crystal structure of 
the KcsA potassium channel was elucidated. One common feature between gA and KcsA is the ion 
selectivity mechanism. In both proteins, the \K\ selectivity is determined by the interaction 
with backbone oxygen atoms and size of the pore (larger ions will find it harder to diffuse 
through the channel)~\cite{Kelkar2007}. However, using gA may be limited as we now know the 
complexity of large ion channels.

Besides being used as a system to study ion conductance, gA is also used as a system to test 
force fields or new computational methods. New methods range from approximate continuum theories 
to highly accurate \textit{ab initio} models. One important example worth noting is the potential 
of mean force (PMF) calculations with non-polarisable force fields of \K\ ion through gA. The 
barrier of the PMF profile reported in the literature ranges between 10--20~kcal/mol for 
\K~\cite{Bastug2006c,Allen2003,Allen2004,Bastug2007}. This is an extremely high energy barrier, 
and \K\ will not be able to diffuse through the pore. Experimental data show that 
\K\ permeates near diffusion rates~\cite{Hille2001}. With this shortcoming, corrections are applied 
to the calculations to arrive at the experimental conductance value. However, this approach is 
not satisfactory as it demonstrates the failure of the current non-polarisable force field to 
model specific systems. Recently, a PMF calculation with the AMOEBA polarisable force field gave 
an improvement compared to the results from non-polarisable force fields~\cite{Peng2016}. The 
PMF profile shows a decrease in the energy barrier to $\sim$4 kcal/mol. With this energy barrier, 
the ion conductivity was calculated to be close to the experimental value without any need for 
additional corrections. This demonstrates that gA can be used as a toy model to test new force 
fields and highlights the need for the explicit treatment of polarisation in classical MD simulations.

A small portion of this thesis deals with the effect of lipid membranes on \K\ ion permeation 
through gA. Experiments showed a significant decrease in ion conductance when gA is embedded 
in phospholipid compared to ceramides~\cite{DeGodoy2011}. However, proton conductance shows 
the opposite behaviour~\cite{Wyatt2009}. This thesis reports on the investigation of \K\ ion 
permeation through gA embedded in two different lipid molecules: POPC and NODS. 

\section{Ion Solvation}
\chapquote{``The cure for anything is salt water: sweat, tears, or the sea."}{Isak Dinesen}

\vskip 0.5cm

Ions in water, i.e. electrolyte solutions, are formed by mixing salt with solvents like water. 
Sodium-chloride salt (NaCl) is a simple example that demonstrates this process. Initially, NaCl 
is in solid form (usually as a salt crystal) and dissociates into \Na\ and \Cl\ ions in water. 
In biomolecular simulations, ions in water are the focus instead of solids. Therefore, an accurate 
description of electrolytes is crucial and a first step in designing and optimising empirical 
force fields. In non-polarisable force fields, the charges on atoms are fixed; thus, only the LJ 
parameters are considered in the optimisation process. Optimising ion parameters usually involves 
fitting the LJ parameters to ion-water radial distribution function (RDF), solvation free energy 
and osmotic/activity coefficients obtained from simulation to available experimental 
data~\cite{Aqvist1990,Jensen2006,Luo2010a}. Most empirical force fields available can reproduce 
these quantities reasonably well for monovalent ions but fails for divalent ions~\cite{Timko2011}. 
This is due to the lack of explicit treatment of polarisation in the force field. Nevertheless, 
getting the right parameters for monovalent ions is more important than divalent ions as they are 
more common in biomolecular processes.

Calculations of ion solvation free energy go back to Max Born with the Born equation~\cite{Born1920}. 
This equation treats the water molecules as a continuum medium, and the ion is a point charge 
located at the origin. For solute molecules with distributed charges, a modification to the 
Born equation is needed. The modification includes higher-order multipole moments to describe 
the interaction of the solute molecule and solvent medium. One equation that includes higher 
moments is the Kirkwood-Onsager equation~\cite{Kirkwood1934}. Since these methods uses a pure 
continuum model, atomistic details are omitted and may not provide accurate results.

Atomistic based simulations of ion solvation free energy started in the 1990s using free energy 
perturbation method. One issue with calculating solvation free energy with atomistic simulations 
is the choice of the boundary. The two boundaries that are used with atomistic simulations: 
periodic boundary condition (PBC)~\cite{Hummer1995,Hummer1996} and spherical boundary condition 
(SBC)~\cite{King1989,Beglov1994,Aqvist1990}. In the former, the ion is surrounded by water 
molecules in a cubic box representing an infinite system. In the latter, the ion is placed in 
a spherical cavity filled with water molecules. Since atomistic simulations are limited in the 
system size that can be simulated, both boundary conditions require a finite-size correction. 
For PBC the correction was derived in Ref.~\cite{Hummer1996} while for SBC, the Born equation is 
used~\cite{Aqvist1990}. The corrections are analytical and are applied \textit{a posteriori}. 
One major difference between the two boundary condition is the lack of a vacuum interface in 
PBC. Thus calculations obtained with each boundary conditions using the same ion parameters 
will result in different values. The difference is a constant energy shift known as the Galvani 
or surface potential~\cite{Lin2014a}. The surface potential can be estimated from simulation but 
is difficult in experiments~\cite{Lamoureux2006}. With this correction, solvation free energies 
calculated with PBC can agree with values obtained with SBC and vice-versa. This is important 
when comparing with experimental data because different methods estimate the proton \Hi\ 
solvation free energy differently. The variations in the literature depend on the assumptions 
made, and some like the cluster approximation method~\cite{Tissandier1998} include a vacuum 
interface while the extra-thermodynamic hypothesis~\cite{Schmid2000,Marcus1994} does not. 

A portion of the research reported in this thesis looks at the systematic study of ion 
solvation free energy with the two boundary conditions. The convergence of the energies with 
system size is reported, and with both boundary conditions, convergence is reached with similar 
volumes. A derivation of a generalised Born equation is derived using the image charge method. 
Application to amino acid side chain analogues reveals overestimation of the solvation free 
energy for aspartate and glutamate residues.

\section{Outline}
\chapref{chap:methods} gives an overview of the theory and computational methods of MD. This 
is followed by an explanation of free energy calculations used in the investigations of this 
thesis. \chapref{chap:gltph} is a comprehensive review of glutamate transporters with regards 
to the computational studies reported in the literature. Following the review on glutamate 
transporters, three chapters report the work on the protein-ligand interactions in \GltPh. 
First, the elucidation of the Na2 site is presented in \chapref{chap:na2}, then the ligand-binding 
mechanism in \GltPh\ is discussed in \chapref{chap:bind}. Finally, the escape of \Na\ 
from the Na3 binding site is given in \chapref{chap:unbind}. The next two chapters report on 
two side projects; ion permeation in gA and ion solvation. The work on gA is discussed in 
\chapref{chap:gA} and deals with \K\ permeation through gA embedded in different lipid 
membranes. The final work is reported in \chapref{chap:ions} that investigates ion solvation 
with SBC. The summary of the research and potential future work is discussed in 
\chapref{chap:conc}. Lastly, the \hyperref[apx:funnel]{Appendix} contains the source code 
of scripts and programs used in analysis and simulations.
%=======================================================================================%