%=======================================================================================%
\chapter{Review of Glutamate Transporters}
\label{chap:gltph}
This chapter gives a comprehensive review of the computational studies reported 
in the literature for glutamate transporters. In particular, the review primarily 
focuses on the aspartate transporter GltPh. More work has been carried out on 
GltPh due to the availability of its crystal structure. The review explains the 
current state of research and problems encountered in the computational studies 
with regards to explaining some of the phenomena observed in experiments. Also, 
included is a brief review of the mammalian EAATs and the neutral transporters 
ASCTs. The particular research question addressed in this thesis is given at the 
end of the chapter. This review is a published material (see Ref~\cite{Setiadi2015}) 
and with a slight modification presented here as new literature is available since 
its publication. Also, the computational methods in the article are removed here 
so that the chapter solely focuses on glutamate transporters. The methods discussed 
in the paper are explained in the previous chapter.

\newpage
\section{Introduction}
Neurons in the central nervous system communicate using organic molecules called 
neurotransmitters. When an action potential arrives at the synapse of a neuron, 
it opens the calcium channels. The increased concentration of \Ca\ ions triggers 
the release of neurotransmitters to the synaptic cleft between two neurons. The 
neurotransmitters diffuse through the cleft and bind to the receptors at the 
neighbouring neuron, some of which are ligand-gated ion channels. Binding of an 
excitatory neurotransmitter such as glutamate to a receptor opens a cation 
channel, which  depolarises the membrane potential and increases the probability 
of generating an action potential~\cite{Lisman2007}. Conversely, binding of an 
inhibitory neurotransmitter such as GABA or glycine to a receptor opens a 
potassium or chloride channel, which hyperpolarises the membrane potential and 
thereby suppresses the formation of an action potential.

%% Why do these proteins matter?
Glutamate is the major excitatory neurotransmitter in the mammalian central 
nervous system \cite{Danbolt2001}. It thus plays a pivotal role in brain 
function and disruption of the processes involving glutamate results in a 
number of neurological disorders. Glutamate molecules are loaded in synaptic 
vesicles by vesicular glutamate transporters and released to the synaptic cleft 
through the action of SNARE proteins~\cite{ElMestikawy2011}.  The extracellular 
concentration of glutamate is in the nanomolar range while in the cytoplasm, it 
is in the millimolar range~\cite{Herman2007}. Excess glutamate in the synaptic 
cleft results in over-activation of receptors---called excitotoxicity---which 
leads to neuronal damage and eventual cell death. Such effects have been 
associated with Alzheimer's disease \cite{Hynd2004}, amyotrophic lateral 
sclerosis \cite{Rothstein1992}, ischaemia \cite{Rossi2000} and epilepsy 
\cite{During1993}. Thus rapid removal of excess glutamate from the synaptic 
cleft is essential for normal functioning of neurons.

%% EAATs and ASCTs brief description
The concentration of glutamate in the synaptic cleft is maintained by specific 
transport proteins called excitatory amino acid transporters (EAATs) 
\cite{Danbolt2001}. A cartoon representation of the transport mechanism is   
shown in \figref{review:fig1}, which involves two half-cycles. In the first 
part, three \Na, one \Hi, and Glu are bound to the outward-facing state and 
translocated across the membrane. In the second part, this cargo is released 
to the cytoplasm, followed by binding of a \K\ ion which is counter-transported 
to the outward-facing state and released to the plasma, completing the cycle
\cite{Zerangue1996,Levy1998,Owe2006}.

\begin{figure}[t!]
\centering
\includegraphics[width=0.6\textwidth]{Figures/Review/fig1.jpg}
\caption{Mechanism of coupled-glutamate transport in EAATs. Step 2 shows the 
binding of the Na2 ion, which occurs after the binding of the substrate and the 
closure of the HP2 gate. Step 5 shows the opposite happening in the 
inward-facing state. Steps 3-4 correspond to the translocation of the transport 
domain across the membrane with 3 \Na, \Hi, and Glu bound to 
EAAT, while steps 7-8 depicts the same with only \K\ bound. }
\label{review:fig1}
\end{figure}

EAATs are part of the solute carrier family 1A, which includes neutral amino 
acid transporters called Alanine-Serine-Cysteine transporters (ASCTs) and other 
prokaryotic transporters \cite{Slotboom1999}. There are five known subtypes of 
EAATs (EAAT1-5) and two ASCTs (ASCT1-2) 
\cite{Arriza1993,Shafqat1993,Utsunomiya-Tate1996}. Both EAATs and ASCTs couple 
the substrate transport to the concentration gradient of \Na\ ions, and hence 
belong to the class of secondary active transporters. Within the EAATs family, 
the subtypes share 50-60\% homology and are known to have the same transport 
mechanism. In comparison, ASCTs share 30-40\% homology to EAATs and employ a 
simpler transport mechanism. The transport is independent of \Hi\ and \K\ 
ions, which is similar to the aspartate transporter \GltPh, but 
the number of \Na\ ions required for transport has not been established yet.

%% Crystal Structure
A major breakthrough in the study of glutamate transporters was the solution of
the crystal structure of the prokaryotic homolog \GltPh\ from
\textit{Pyrococcus horikoshii} in the outward-facing conformation
\cite{Yernool2004}. Since then, successive iterations of the crystal structure
of \GltPh\ have been resolved, including the binding sites of the substrate and
two sodium ions \cite{Boudker2007}, the inward-facing \cite{Reyes2009} and
intermediate \cite{Verdon2012} conformations. Following the argument of
reductionism, \GltPh\ provides a relatively simpler model for EAATs and
ASCTs. \GltPh\ shares 36\% amino acid sequence identity with EAATs
\cite{Yernool2004} and 23\% with ASCTs \cite{Shafqat1993}. Although the sequence
identity is low for the overall protein, the sequence identity in the binding
pocket is as high as 60\%. \GltPh\ differs from EAATs in its transport cycle in
that it does not require the co-transport of \Hi\ or the counter-transport of
\K, and is selective for aspartate \cite{Ryan2009}. For a thorough
discussion of the experimental work on \GltPh\ and its relevance to EAATs,
we refer to a recent review article \cite{Vandenberg2013}.

%% Computational Investigations and review
Determination of the crystal structure of \GltPh\ has opened the way for
computational investigation of glutamate transporters. Many atomistic molecular
dynamics (MD) and coarse-grained simulations of \GltPh\ have been performed to
obtain  further insights on the mechanism and energetics of aspartate/glutamate
transport. In this review, we focus on the computational work involving the
crystal structure of \GltPh. In particular, we discuss how computational methods
are used to model and probe the transport mechanism and energetics that are
otherwise difficult to access using experimental techniques. In addition to
the work on \GltPh, recent computational efforts to construct homology models
of EAATs and ASCTs based on the \GltPh\ structure are discussed. 

\section{MD Simulations of Glutamate Transporters}
\subsection{Prokaryotic Homologue \GltPh}
% Structure of GltPh
\GltPh\ exists as a trimer in the membrane with three identical subunits held 
together by non-covalent bonds. Within each subunit (or monomer) there are eight 
transmembrane segments (TM1--TM8) and two hairpin loops (HP1 and HP2) between 
TM6 and TM8. In the outward state, the trimer has a bowl-shaped structure with 
a diameter of about 50~\angs\ and depth 30~\angs. The trimer can be grouped into 
two domains: trimerisation and transport. The trimerisation domain consists of 
TM1, 2, 4 and 5 from each subunit, whose function is to stabilise the trimer 
complex. The transport domain consists of TM3, 6, 7 and 8 along with the two 
hairpins, HP1 and HP2 from each subunit. The structure of the transport domain 
of \GltPh\ with the bound aspartate and three \Na\ ions is shown in 
\figref{review:fig2}.

\begin{figure}[t!]
\centering
\includegraphics[width=0.6\textwidth]{Figures/Review/fig2.jpg}
\caption{Transport domain of a subunit of \GltPh\ depicting the 
positions of Asp and three \Na\ ions (green spheres numbered from 1 to 3). The 
TM segments and hairpins involved in the coordination of Asp and \Na\ ions 
are indicated as follows: HP1 (yellow), HP2 (red), TM3 (blue), TM7 (orange), 
and TM8 (magenta). The upward motion of HP2 opens the outward gate and exposes the 
ligands to water.}
\label{review:fig2}
\end{figure}

% Position of the 3rd Na ion
The crystal structure of \GltPh\ resolved in 2007 identified the binding sites 
for the substrate and two \Na\ ions labelled Na1 and Na2~\cite{Boudker2007}. 
As the stoichiometry of ion coupling was not known at that time, initial MD 
simulations of \GltPh\ were performed using this structure 
\cite{Shrivastava2008,Huang2008}, where the opening of the HP2 gate was 
demonstrated to be the outward gate. Subsequent radio-labelled experiments with 
$^{22}$\Na\ ions indicated that three \Na\ ions are co-transported in 
\GltPh~\cite{Groeneveld2010}. Several binding sites were proposed for the 
third \Na\ ion (labelled Na3) from  computational results and experimental 
observations~\cite{Holley2009,Larsson2010,Tao2010,Huang2010,Bastug2012}. 
According to the mutagenesis experiments on EAAT3~\cite{Tao2010,Tao2006}, the 
side chains of residues T92 and D312 are involved in the coordination of one of 
the co-transported \Na\ ions during transport. Because neither residues 
coordinate Na1 or Na2 in the crystal structure of \GltPh~\cite{Boudker2007}, 
the side chains of these residues indicate a possible binding site for Na3. The 
observation strengthened this argument that in MD simulations of \GltPh\ 
with Na1 and Na2 ions, the D312 side chain was observed to flip and start 
coordinating Na1~\cite{Larsson2010,Bastug2012}. The resulting coordination 
shell for Na1 is in conflict with the crystal structure of \GltPh, underscoring 
the need for a cation at the site of the D312 side chain to prevent its flipping.  
Such a Na3 binding site was first proposed from electrostatic calculations, 
consisting of the side chains of Y88, T92, N310 and D312, and the backbone of 
G404~\cite{Tao2010}. Refinement of this site in MD simulations using the closed 
structure of \GltPh\ resulted in a slightly different Na3 site, with the side 
chains of T92, N310 and D312 retained but Y88 and G404 replaced by a water 
molecule~\cite{Huang2010}. Yet a third Na3 site was proposed from the MD 
simulations of TBOA-bound open structure of \GltPh, where the side chains of 
T92, N310 and D312 were again retained, but the coordination shell was completed 
with the side chain of S93 and the backbone of Y89. The difference between the 
two MD simulations is caused by the flipping of the N310 side chain between the 
closed and open structures of \GltPh. Mutagenesis experiments (in particular, 
Y89A~\cite{Bastug2012}) indicated that the last proposed site was the most 
likely site for binding of Na3. This prediction is verified with the recent 
crystal structure of \GltTk\ that shows all three \Na\ and aspartate bound. 
Here, \Na\ ion at Na3 is bound by Y89, T92, S93, N310 and D312 and is also 
stable in MD simulations~\cite{Guskov2016}.

% Binding of ligands
With the binding site for Na3 established, the \GltPh\ model was ripe for
free energy calculations for binding of substrate and \Na\ ions. Such 
calculations were performed first for the outward-facing conformation of 
\GltPh\ \cite{Heinzelmann2011}, followed by calculations for the inward-facing 
conformation \cite{Heinzelmann2013}. The results summarised in 
\tabref{review:tab1} show that
\begin{enumerate}[(i)]
    \item the order of binding of the ligands in the outward-facing conformation 
    is Na3, Na1, Asp, (Outward gate closes), and Na2 (the Na2 site forms only 
    after HP2 gate closes);
    \item the binding free energies of the ligands are very similar in the 
    outward- and inward-facing conformations, consistent with the observation 
    that the binding pockets are preserved in the corresponding crystal structures;
    \item release of the ligands in the inward-facing conformation follows the 
    reverse of the order of binding, that is, Na2, (Inward gate opens), Asp, Na1, 
    and, Na3.
\end{enumerate}

\begin{table}[t!]
\caption{Ligand binding free energies, $\Delta G_{b}$, of \Na\ and Asp
in \GltPh\ for both outward- and inward-facing conformations (units, 
kcal/mol). The ligands present during the free energy calculations are 
indicated in parenthesis (other combinations of ligands yield higher free 
energies, and therefore are not shown) \cite{Heinzelmann2011,Heinzelmann2013}.}
\label{review:tab1}
\begin{center}
\begin{tabular}{lcc}
\hline
Ligand & Outward & Inward \\ \hline
Na3 & -18.7 $\pm$ 1.1 & -16.3 $\pm$ 1.1 \\
Na1 (Na3) & -7.1 $\pm$ 1.3 & -7.3 $\pm$ 1.3 \\
Asp (Na1, Na3) & -3.8 $\pm$ 1.0 & -4.9 $\pm$ 1.1 \\
Na2 (Na1, Na3, Asp) & -2.7 $\pm$ 1.3 & -2.4 $\pm$ 1.2 \\ \hline
\end{tabular}
\end{center}
\end{table}

% Na2
We note that the Na2 binding site used in the free energy calculations is 
slightly shifted from its position observed in the crystal structure 
\cite{Boudker2007}. MD simulations show that \Na\ is unstable in the Na2 site, 
and one explanation for this is that the proposed site in the crystal structure is 
not the actual Na2 site. The suspicion arises due to the use of \Tl\ instead of \Na\ 
in resolving the crystal structure~\cite{Boudker2007}. Unfortunately, probing the 
Na2 site with mutagenesis experiments is very difficult because the coordinating atoms 
are mostly backbone carbonyl oxygens. Na2 is bound to TM7 and HP2 carbonyl oxygen atoms, 
and is exposed to water. Thus the binding site is formed only after the HP2 gate closes, 
and binding of Na2 is proposed to lock this gate~\cite{Boudker2007}. Presence of Na2 
is expected to prevent water molecules disrupting the hydrogen bonds between HP1 
and HP2 and thereby open the gate. While the calculated Na2 binding free energy 
is quite small, conformational changes occurring after the gate closure may 
lead to a more stable Na2 binding site~\cite{Lev2013}. The effect of the substrate 
transport on the Na2 site was also investigated in a recent steered MD study 
\cite{Venkatesan2015}. The simulation results indicated maturing of the Na2 site 
proposed earlier \cite{Heinzelmann2011} during the substrate transport, which was 
validated in mutation experiments \cite{Venkatesan2015}. The \Na\ ion at this site 
is coordinated by the side chain of T308 and is the site used in the free energy 
calculations in \tabref{review:tab1}.

An important issue clarified by the free energy calculations is the coupling 
between the substrate and the \Na\ ions. In some secondary transporters (e.g., 
LeuT), a \Na\ ion is in direct contact with the substrate so the coupling 
mechanism is obvious and the strength is assured. In \GltPh, however, the 
closest \Na\ ion to Asp is Na1, which is 7~\AA\ away from the nearest carbonyl 
oxygen of Asp, separated by a water molecule and the side chain of S278 
(\figref{review:fig3}). Nevertheless, the hydrogen-bond network induced by 
Na1 appears to be sufficient to stabilise Asp in the binding site. Conversely, 
removal of Na1 leads to the disruption of the hydrogen-bond network and Asp is 
released to the solvent in a few ns \cite{Heinzelmann2011}. Because of the fast 
release of Asp in the absence of Na1, it was not possible to estimate the 
Asp--Na1 coupling free energy in the outward-facing conformation. A more stable 
Asp in the inward-facing conformation allowed such a calculation, and the 
presence of Na1 was found to change the binding free energy of Asp from 
$+1.8$~kcal/mol to $-4.9$~kcal/mol \cite{Heinzelmann2013}. Thus the presence of 
Na1 is essential for binding of Asp with the Asp--Na1 coupling contributing 
$-6.7$~kcal/mol to its binding free energy.

\begin{figure}[t!]
\centering
\includegraphics[width=0.6\textwidth]{Figures/Review/fig3.jpg}
\caption{The binding pocket of \GltPh\ in the open state of the 
outward-facing conformation, showing the hydrogen-bond network (purple dotted 
lines) that couples Na1 (yellow sphere) to Asp (green backbones).}
\label{review:fig3}
\end{figure}

It is important to stress that the results in \tabref{review:tab1} are based on 
the crystal structures and do not take into account any conformational changes 
that may occur during the ligand binding or release processes~\cite{Hanelt2015}. 
For example, the path to Na3 goes through the Na1 site and is blocked by a 
hydrogen-bond network in the apo state~\cite{Huang2010}. Thus binding of the 
first two \Na\ ions are expected involve some conformational change and may 
even be coupled to facilitate the breaking of the hydrogen-bond network. 
This could be the reason for the wrong prediction in simulations with that of 
experiments for the \Na\ and Asp binding free energy. \tabref{review:tab1} shows 
a high-affinity and low-affinity binding free energy for Na1 and Asp, respectively, 
where experiments demonstrate the opposite~\cite{Hanelt2015,Ewers2013,Reyes2013b}.
In the case of ligand release, however, there is less scope for such a coupling, 
and the very low binding free energy predicted for Na3 is likely to stand. Then 
unbinding of Na3 is expected to be the rate-limiting step in the transport cycle. 
This is supported by the mutation of the residues coordinating Na3, which reduces 
its binding affinity and thereby increase the transport rate by 20-fold. Estimates 
for the release time of Na3 obtained from Kramer's rate theory give consistent 
results with the experimental turn over rates in \GltPh, providing further support 
for this hypothesis~\cite{Heinzelmann2013}.

Conformational changes that occur in \GltPh\ during the transport cycle can be 
divided into three groups: i) gating motions and ligand binding in the 
outward-facing state, ii) transition of the transport domain from the 
outward-facing to the inward-facing state, iii) gating motions and ligand 
release in the inward-facing state. The gating motions and substrate 
binding/release in (i) and (iii) are relatively fast and localised events while 
the transition in (ii) is slow and highly non-local. Therefore, the former has 
been studied using MD and metadynamics simulations, but the latter had to be 
studied using coarse-grained methods such as anisotropic network model (ANM) 
\cite{Bahar2010}. 

The unique role of HP2 in the opening of the outward gate was well established from 
the early MD simulations \cite{Shrivastava2008,Huang2008}. From symmetry 
arguments, HP1 was initially proposed as the inward gate 
\cite{Reyes2009,Crisman2009}, and this was partially supported from metadynamics 
simulations where a larger movement of HP1 compared to HP2 was predicted 
\cite{Grazioso2012}. However, the opposite has been observed in unbiased MD 
simulations of the inward-facing state; namely, HP1 moves little and preserves 
its contacts with the substrate while HP2 moves further and loses some contacts 
with the substrate \cite{DeChancie2011a,Zomot2013,Heinzelmann2013}. In 
particular, the multiple microseconds MD simulations performed in the last work 
\cite{Zomot2013} has clearly established the dominant role of HP2 as the inward  
gate. The relatively smaller opening of HP2 in the inward gate compared to the 
outward gate happens because HP2 is partially buried inside the protein in the 
former case and thus has less scope to move.
In metadynamics \cite{Grazioso2012} and long MD \cite{Zomot2013} simulations, 
the release of Na2, Asp, and Na1 to the cytoplasm was also observed, and the order 
of release was consistent with that predicted from free energy calculations. 
Unfortunately, Na3 was not included in these simulations. Clarification of the 
role of Na3 in limiting the rate of transport will be of great interest 
through independent computational studies.

Computational investigation of the outward $\to$ inward transition of the 
transport domain using MD simulations is much more demanding and has been 
rarely attempted. For example, a steered MD simulation of this transition did 
not yield satisfactory results presumably due to insufficient sampling~\cite{Gu2009}. 
Instead, progress has been made using ANMs in combination with MD 
simulations~\cite{Jiang2011,Stolzenberg2012,Gur2013,Das2014}. These studies have 
shown that the transition occurs independently in each subunit involving large 
scale collective motion of the binding pocket~\cite{Jiang2011}, and the 
transport and trimerisation domains move in opposite directions along the 
membrane normal~\cite{Stolzenberg2012}. Use of ANM to study large scale domain 
motions was justified through a comparison of the results to long MD simulations~\cite{Gur2013}. 
Finally, the transition pathway of the transport domain between the outward and 
inward states was constructed using a two-state ANM~\cite{Das2014}. Complementary 
experimental work has been done using single-molecule FRET imaging, and indicate 
that each subunit in the trimer moves independent of the others and the transition 
is stochastic~\cite{Erkens2013,Akyuz2013,Akyuz2015}.

% Cl conductance
A common feature of the glutamate transporter family is the existence of a 
\Cl\ channel within the transporter~\cite{Vandenberg2013}. The \Cl\ channel 
is conserved among glutamate transporters, indicating its importance in the 
functioning of the transporter. The \Cl\ channel is activated by the binding 
of the substrate and \Na\ ions but is uncoupled from the transport cycle as 
demonstrated by the mutation of residues in EAATs, which block the substrate 
transport but does not impede \Cl\ conductance~\cite{Fairman1995}. So far 
there are no crystal structures of \GltPh\ with an open \Cl\ channel, which 
has impeded the computational study of \Cl\ conductance. Very recently, a model 
of \GltPh\ with an open \Cl\ channel has been constructed~\cite{Machtens2015}, 
using the experimental observation that an aqueous cavity partly forms the 
channel at the interface of the transport and trimerisation domains~\cite{Cater2014}. 
In this study~\cite{Machtens2015}, a membrane potential of 
$\pm 1.6$~V was applied to both the outward- and inward-facing states of 
\GltPh\ in order to create such a cavity, but an open channel conformation 
was not found during 8~$\mu$s MD simulations. This suggested that the \Cl\ 
channel may be formed in an intermediate state of the transporter during 
substrate translocation. Intermediate states of \GltPh\ were then generated 
using essential dynamics sampling---a rare event sampling method. Application 
of a membrane potential in an intermediate state indeed resulted in separation 
of the trimerisation and transport domains, creating a water-filled open channel 
conformation. The proposed model~\cite{Machtens2015} is mostly consistent with 
experimental observations, e.g., the predicted pore diameter is 5.6~\AA\ while 
the experimental values are in the range 5--6 \angs~\cite{Vandenberg2013}, and 
the R276S mutation at the centre of the pore converts it to a \Na\ channel as 
observed experimentally~\cite{Borre2004b}.

\subsection{Excitatory Amino-Acid Transporters}
There is now one crystal structure of EAATs available in the protein data bank. 
This structure is based on an engineered EAAT1 (dubbed EAAT1\textsubscript{cryst}) 
whose sequence at the end of TM3 and TM4 is replaced with that of ASCT2 to increase 
the ligand conductance~\cite{Canul-Tec2017}. However, this structure was resolved in 
2017 and all of the work on EAATs, as of now, has been on homology models based on 
the \GltPh\ structure. While the overall sequence identity between \GltPh\ and EAATs 
is low, it is over 60\% in the binding pocket. Thus homology models may be able to 
explain the functional differences between \GltPh\ and EAATs, namely, co-transport 
of \Hi\ and counter-transport of \K\ ion. To highlight the functional similarity 
between \GltPh\ and EAATs, we compare in \tabref{review:tab2} all the important 
residues in the binding pocket identified from the crystal structures and mutation 
experiments. Focusing on EAAT3 as there are more data on it, we see that only five 
residues differ from \GltPh. Of these, R276 and T352 in \GltPh\ contribute backbone 
carbonyls to the coordination of the substrate and Na2, respectively, and the other 
three are not involved in the coordination of the ligands. Thus we expect the 
coordination shells of the \Na\ ions and the substrate to be preserved. Moreover, the 
mutations, R276 $\to$ S331 and M395 $\to$ R445, transfer the position of arginine 
from HP1 in \GltPh\ to TM8 in EAAT3 but do not change the location of its side chain. 
Thus they preserve the structural similarity. The only other significant mutation 
in \tabref{review:tab2} is Q318 $\to$ E374. The E374 residue has been proposed to be 
the protonation site in EAAT3 from mutation experiments---the E374Q mutation does 
not affect the Glu affinity but abolishes the pH dependence of Glu transport 
\cite{Watzke2000,Grewer2003b}.

\begin{table}[b!]
\caption{\GltPh\ residues involved in the coordination of the ligands
and their equivalents in EAAT1, EAAT2 and EAAT3 (human sequences are used). 
Residues from the mutagenesis experiments are also included in the table. 
The residues that are not conserved between \GltPh\ and EAATs are indicated 
with red.}
\label{review:tab2}
\begin{center}
\begin{tabular}{ l l c c c c c c c c c}
\hline
Glt$_{\rm Ph}$ 
&Y89&T92&S93&\textcolor{red}{Q242}&\textcolor{red}{R276}&S277&S278&G306&T308 \\ \hline
EAAT1 
&Y127&T130&T131&\textcolor{red}{H328}&\textcolor{red}{S363}&S364&S365&G394&T396
\\
EAAT2 
&Y124&T127&T128&\textcolor{red}{H326}&\textcolor{red}{A361}&S362&S363&G392&T394
\\
EAAT3 
&Y98&T101&T102&\textcolor{red}{H296}&\textcolor{red}{S331}&S332&S333&G362&T364
\\ \hline
Glt$_{\rm Ph}$ 
&N310&D312&T314&Y317&\textcolor{red}{Q318}&S349&I350&\textcolor{red}{T352}&G354 
\\ \hline
EAAT1 
&N398&D400&T402&Y405&\textcolor{red}{E406}&S437&I438&\textcolor{red}{A440}&G442
\\
EAAT2
&N396&D398&T400&Y403&\textcolor{red}{E404}&S435&I436&\textcolor{red}{A438}
&\textcolor{red}{S440} \\
EAAT3 
&N366&D368&T370&Y373&\textcolor{red}{E374}&S405&I406&\textcolor{red}{A408}
&G410 \\ \hline
Glt$_{\rm Ph}$  &V355&G359&D390&D394&\textcolor{red}{M395}&R397&T398&N401&D405\\ \hline
EAAT1  &I443&G447&D472&D476&\textcolor{red}{R477}&R479&T480&N483&D487 \\
EAAT2  &I441&G445&D470&D474&\textcolor{red}{R475}&R477&T478&N481&D485 \\
EAAT3  &V411&G415&D440&D444&\textcolor{red}{R445}&R447&T448&N451&D455 \\
\hline
\end{tabular}
\end{center}
\end{table}

Homology models for EAAT3 in the outward- and inward-facing states have been 
recently constructed using the available crystal structures of \GltPh\ 
\cite{Heinzelmann2014b}. MD simulations of these EAAT3 models have revealed 
that:
\begin{enumerate}[(i)]
    \item Glu is stably bound when E374 is protonated but becomes unstable when 
    E374 is deprotonated in both the outward and inward-facing states. Thus 
    binding of Glu is contingent upon the protonation of E374, which is consistent 
    with the experimental observations indicating E374 as the protonation site 
    \cite{Watzke2000,Grewer2003b}.
    \item The coordination shells for Na1, Na3, and Glu are very similar to those 
    in \GltPh, consistent with the expectations from the alignment diagram in 
    \tabref{review:tab2}. The Na2 coordination is somewhat different from that of 
    \GltPh\ in that S405 carbonyl, and T364 hydroxyl are not involved in the 
    coordination of Na2, and Na2 is not stably bound. It is possible the Na2 site 
    is not conserved in EAATs. Further experimental and computational work is 
    required to determine the Na2 binding site in EAATs.
    \item Gating in the outward-facing state is very similar to that in \GltPh, 
    but a rather different mechanism occurs in the inward-facing state---HP1 and 
    HP2 move about similar amounts, leading to a much larger opening of the gate 
    compared to that in \GltPh. This can be traced to the transfer of an
    arginine (R276) from HP1 in \GltPh\ to TM8 in EAAT3 as mentioned above. In 
    \GltPh, R276 forms a salt bridge with D394, which prevents the opening of 
    HP1. Transfer of this arginine (R445) to TM8---which still makes a salt 
    bridge with D444---enables the larger opening of HP1, facilitating the release of 
    the larger Glu substrate. \figref{review:fig4} illustrates the conformation
    of the gate in EAAT3.
    \item A number of sites have been proposed for binding of a \K\ ion in the 
    inward-facing state of EAATs. The most likely \K\ sites, together with the 
    complete coordination shells obtained from the MD simulations of the EAAT3 
    model, are listed in \tabref{review:tab3}. Site 1 is very similar to the Na1 
    site and is proposed because the mutations that turn the transporter into 
    an exchanger (e.g., D455N) also impair its interaction with a \K\ ion 
    \cite{Teichman2009,Mwaura2012}. Site 2 corresponds to the proton binding 
    site---when the proton leaves, \K\ could bind there to neutralise the 
    site \cite{Kavanaugh1997}. The last site overlaps with the substrate 
    $\alpha$-amino group and was predicted from electrostatic mapping calculations 
    \cite{Holley2009}. To assess the likelihood of each site being the \K\ site, 
    binding free energies and the \K/\Na\ selectivity free energies were 
    calculated (\tabref{review:tab3}). The selectivity free energies were 
    calculated to check the hypothesis that the last \Na\ ion is exchanged with 
    a \K\ ion in EAATs in order to speed up the very low transport rates observed 
    in \GltPh\ \cite{Heinzelmann2014b}. Site 1 appears to be the most likely \K\ 
    site as it has the largest affinity for \K. Also, it is consistent with the 
    \K--\Na\ exchange hypothesis as it has negligible \K/\Na\ selectivity. 
    Recent crystal structure of \GltTk---a close homologue of \GltPh\ resolved in 
    the apo state--- provides further experimental support for this site 
    \cite{Jensen2013}. The \GltTk\ structure exhibits some conformational 
    differences from that of \GltPh\ in the vicinity of site 1, 
    which are well reproduced in the computational model. 
\end{enumerate}

\begin{figure}[t!]
    \centering
    \includegraphics[width=1.0\textwidth]{Figures/Review/fig4.jpg}
    \caption{Superposed structures of EAAT3 showing the opening of the gate
    in the (A) outward- and (B) inward-facing conformations. Three \Na\ 
    and the glutamate substrate are also shown in the figure. The colour grey
    represents the closed conformation and purple represents the open 
    conformation. The figure is obtained from Ref.~\cite{Heinzelmann2014b}.}
    \label{review:fig4}
\end{figure}

\begin{table}[b!]
\caption{EAAT3 residues coordinating the \K\ ion at three proposed binding
sites, and their respective standard binding free energies and \K/\Na\ 
selectivity free energies (in units of kcal/mol). }
\begin{center}
\begin{tabular}{llll}
\hline
 & Site 1 & Site 2 & Site 3 \\ \hline
\multirow{7}{*}{Helix-Residue} & \multicolumn{ 1}{l}{TM7 - G362 (O)} & TM7 - 
T370 (OH) & HP1 - S331 (O) \\
\multicolumn{ 1}{l}{} & TM7 - I365 (O) & TM7 - T370 (O) & HP1 - S331 (OH) \\
\multicolumn{ 1}{l}{} & TM7 - N366 (O) & TM7 - E374 (O\textsubscript{1}) & TM8 - 
D444 (O) \\
\multicolumn{ 1}{l}{} & TM8 - D455 (O\textsubscript{1}) & TM7 - E374 
(O\textsubscript{2}) & TM8 - D444 (O\textsubscript{1}) \\
\multicolumn{ 1}{l}{} & TM8 - D455 (O\textsubscript{2}) & H2O (1) & TM8 - D444 
(O\textsubscript{2}) \\
\multicolumn{ 1}{l}{} & H2O (1) & H2O (2) & TM8 - T448 (OH) \\
\multicolumn{ 1}{l}{} & H2O (2) &  & H2O \\ \hline
\multicolumn{ 1}{l}{$\Delta G_{b}$ (\K)} & -20.5 $\pm$ 1.1 & -9.5 $\pm$ 1.2 
& -6.5 $\pm$ 0.8 \\
\multicolumn{ 1}{l}{$\Delta G_{sel}$ (\K/\Na)} & 0.5 $\pm$ 0.4 & 3.9 
$\pm$ 
0.4 & -3.1 $\pm$ 0.4 \\ \hline
\end{tabular}
\end{center}
\label{review:tab3}
\end{table}

The EAAT3 model discussed above \cite{Heinzelmann2014b} confirms that the 
\GltPh\ structure provides a working model for functional studies of EAAT3. 
Furthermore, it provides a rationale for \K\ counter-transport through the 
\K--\Na\ exchange mechanism, which speeds up the transport cycle in EAATs 
compared to \GltPh. It also explains the need for \Hi\ co-transport to keep 
the charge content the same as in \GltPh, that is, the fully bound EAAT3 with 
protonated E374 in the outward-facing state and the \K\ bound EAAT3 with 
deprotonated E374 in the inward-facing state have exactly the same charge 
content as the corresponding states in \GltPh. Thus the energetic cost of the 
ion-coupled substrate transport across the membrane in EAAT3 remains the same 
as in \GltPh. 

In a subsequent paper, this EAAT3 model was used in a detailed study of the 
proton transport mechanism through \pka\ calculations 
\cite{Heinzelmann2014,Bahar2014}. \pka\ of all the acidic residues in the 
transport domain were calculated in the fully-bound outward-facing state of 
EAAT3. E374 was found to be the only acidic residue with a high enough \pka\ 
value to be protonated, confirming it as the protonation site. \pka\ values of 
E374 in different states of EAAT3 were then calculated to get a better 
understanding of the  proton transport mechanism. In the outward-facing state, 
the \pka\ value of E374 was 11.7 in the apo state, went down to 6.6 after the 
binding of Na1 and Na3, went up to 10.4 after Glu binding, and to 19.1 after the 
gate closure. These \pka\ values, in combination with the fact that Glu binds 
after Na1 and Na3, and Glu is not stable with a deprotonated E374, suggest that 
proton and Glu binding are mutually coupled and occur simultaneously. The 
closure of the gate completely isolates E374 from the solvent, leading to the 
dramatic rise in the \pka\ value. Similar \pka\ values of E374 were obtained 
in the inward-facing state, which meant that the E374 would not be deprotonated 
in the apo state. Placement of a \K\ ion in the three proposed sites 
(\tabref{review:tab3}) with a protonated E374 suggested a possible proton release 
mechanism. The \pka\ values with a \K\ placed in sites 1 and 3 were reduced from 
11.7 in the apo state to 4.7 and 6.9, respectively, and the \K\ ion in site 2 
was stable only with a deprotonated E374. These observations indicate that 
binding of a \K\ ion is required for the deprotonation of E374 and completion 
of the transport cycle. Thus, in the most likely scenario, the proton and Glu 
are released simultaneously, and the exchange of the last \Na\ ion with \K\ 
prevents protonation of E374.

\subsection{Neutral Amino-Acid Transporters}
ASCTs transport small neutral amino acid across the plasma membrane and belong 
to the same solute carrier family as \GltPh\ and EAATs. ASCT1 is selective to 
alanine, serine and cysteine as the acronym suggest, however, it can also 
transport threonine, asparagine and proline. ASCT2, on the other hand, have a 
broader selectivity profile and plays an important role in glutamine regulation. 
Very little computational work has been done on ASCTs because there are no 
crystal structures and a relatively smaller number of mutation studies are 
available compared to EAATs to guide modelling efforts. Originally coupling 
of a single \Na\ ion to the substrate was proposed for the transport cycle 
\cite{Broer2000,Grewer2004}. However, the coupling of three \Na\ ions to Glu/Asp 
in EAATs and \GltPh\ suggests that two \Na\ ions may be required for 
efficient transport of neutral amino acids. The similarity of the sequences of 
ASCTs with \GltPh\ in the transport domain and the preservation of the 
three \Na\ binding sites found in \GltPh\ provide further support for this 
argument. In a recent study, the \Na\ dependence of the anion current in ASCT2 
was found to be biphasic, which suggests binding of at least two \Na\ ions to 
ASCT2 \cite{Zander2013}. In this work, MD simulations were also performed on 
a homology model of ASCT2---three \Na\ ions and Ala placed at the ligand-binding 
sites as found in \GltPh\ remained stably bound for 8~ns. Unfortunately, these 
MD simulations are too short for an adequate sampling of the phase space, and 
need to be extended and supported by free energy calculations to determine the 
stoichiometry of the \Na\ coupling in ASCTs. 

% Amanda's paper
In a more recent study, the \Na\ sites in ASCT1 were investigated using both 
mutagenesis experiments and MD simulations \cite{Scopelliti2014}. Mutation of 
the aspartate residues coordinating the \Na\ ions at either the Na1 or the Na3 
site to alanine affected the \Na\ affinity and diminished the substrate 
exchange but did not stop it. This suggests that binding of a \Na\ ion at 
either site is sufficient for the transport of the substrate albeit at much 
reduced rates. In MD simulations of the homology model of ASCT1, all three 
\Na\ sites from \GltPh\ were included in their respective sites in ASCT1, and 
Asp was replaced with Ser. The system was found to be stable during 20~ns of 
simulations. Again this is too short to conclude that ASCT1 can stably bind 
three \Na\ ions. Further MD simulations of ASCT1 and its mutated forms were 
performed for various configurations of ligands and mutations---including an 
additional Na1\prim\ site, which corresponds to the site obtained in MD 
simulations of \GltPh\ when Na3 was not included \cite{Bastug2012}. The 
results obtained from 12 MD simulations were mostly consistent with the 
mutation experiments.

\section{Problem Statement}
Determination of the crystal structures of \GltPh---a prokaryotic homologue of 
glutamate transporters---ushered a new age in studies of glutamate transporters 
where the focus is on a molecular-level understanding of the transport process. 
Computational methods will play a critical role in this effort, as many steps in 
the transport process are not directly accessible by experiments. Already many 
computational studies of \GltPh\ have been performed, leading to a better 
understanding of the transport mechanism. However, there are still some open 
questions on \GltPh. A major portion of this thesis deals with answering some 
of these questions in \GltPh\ and its implications in EAATs. In particular, 
three issues reported in the literature are investigated:
\begin{enumerate}[(i)]
\item The discrepancy between simulations and experiments for the Na2 binding 
      site. In almost all simulations, \Na\ drifts away from the Na2 site as 
      determined from crystallography \cite{DeChancie2011a,Heinzelmann2011,
      Heinzelmann2013,Heinzelmann2014a,Venkatesan2015}. 
\item The apparent ligand binding paradox in \GltPh\ where experiments suggest 
      a low-affinity \Na\ and high-affinity Asp binding \cite{Ewers2013,Hanelt2015} 
      while the opposite is obtained in simulations \cite{Heinzelmann2011,Heinzelmann2013}.
\item The escape path of the last \Na\ in \GltPh\ and the observed conformational changes
      upon ligand binding in observed in experiments \cite{Hanelt2015}.
\end{enumerate}

Through the use of free energy methods (\secref{sec:freenergy}), discrepancies 
between simulations and experiments, as mentioned above, are elucidated. The 
problems are addressed in the next three chapters in the order as listed. 
